### ISSUE TITLE
Question about a use case

### ISSUE THREAD

**Issue by filipe-silva** [firefly-iii/firefly-iii#1570]
Hello!
I'm planning on building a desktop app to use the rest api and insert my bills and other receipts on a semi automated way. 
This little app will basically do OCR and insert what it can, while asking what it can't.
This will have me scan all my documents first, my question is: 
Is there a way to keep my images on the server and have a link/url on firefly to see them on demand?

Thanks and best regards.

**Comment by JC5** [comment]
Yes, but only if you link them to a transaction. When you use the API, you can use the [attachments endpoint](http://firefly-iii.readthedocs.io/en/latest/api/attachments.html).

Uploading attachments requires two separate requests: one for creating the attachment, one for adding the data.

There is another URI for downloading an attachment, that could be used to view the file.
